packages/react-avatar/src/AvatarGetInitials.stories.tsx

```diff
@@ -1,19 +1,16 @@
 import * as React from 'react';
-import { Avatar, AvatarProps } from './index';
+import { Avatar } from './index';
 
-const formatter = (name: string, isRTL: boolean) => {
-  return `${name[0]} ${name[name.length - 1]}`;
+export const GetInitials = () => {
```

Comment:
Should we change this to `Initials` and the file name to `AvatarInitials.stories.tsx`?